service-worker.js

```diff
@@ -0,0 +1,23 @@
+---
+---
+
+importScripts('/scripts/sw-toolbox.js');
+
+const VERSION = '{{ site.time }}';
+
+toolbox.cache.name = "Babel-Cache-" + VERSION;
+toolbox.cache.maxEntries = 150;
+toolbox.cache.maxAgeSeconds = 604800;
```

Comment:
Does that mean the user will see the stale data for a week even if the resources change?